### PR TITLE
buildout for local dev and production

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: config/production.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+  "prettify": true,
+  "fingerprint": ["css", "js"],
+  "optimize": [],
```
Comment (celaus) [inline config/production.json:4]: optimize!

File: config/production.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "prettify": true,
```
Comment (celaus) [inline config/production.json:2]: production does not need to be pretty, right? (this might also be cancelled out by optimize...?)

**Comment by celaus** [comment]
otherwise, ACK

File: config/production.json (hunk 2 above)
Comment (chaudum) [inline config/production.json:2]: prettify is for urls

**Pull request merged**